py-polars/polars/dataframe/frame.py

```diff
@@ -4620,7 +4620,7 @@ def sort(
             .collect(_eager=True)
         )
 
-    def sql(self, query: str, *, table_name: str | None = None) -> Self:
+    def sql(self, query: str, *, table_name: str = "self") -> Self:
```

Comment:
This is from the breaking part I believe.

Reply:
No, it just clarifies the default value; there was never really any point in having it be `None` by default, given that `None` just means "self" anyway 😅